.github/workflows/darwin-tests.yaml

```diff
@@ -94,7 +94,7 @@ jobs:
                         --target darwin-x64-ota-provider-${BUILD_VARIANT} \
                         --target darwin-x64-ota-requestor-${BUILD_VARIANT} \
                         --target darwin-x64-tv-app-${BUILD_VARIANT} \
-                        --target darwin-x64-bridge-app-${BUILD_VARIANT} \
+                        --target darwin-x64-bridge-${BUILD_VARIANT} \
```

Comment:
Hrm.  I'm surprised this passed if the target does not exist.  :(

Reply:
This confused me too .... either TV does build a bridge app (doubt it) or we just do not run bridge tests.

Reply:
Right now we do not run bridge tests.

Reply:
I think currently we just log a 'unknown variant'. Fixing that is probably a separate concern.